pkg/bot/teams.go

```diff
@@ -553,5 +553,9 @@ func replaceEmojiTagsWithActualOne(content string) string {
 
 // emojiMapping holds mapping between emoji tags and actual ones.
```

Comment:
please do me a favour and fix bug on `main`

Constructor:
```
	longFormatter := interactive.NewMDFormatter(longLineFormatter, interactive.MdHeaderFormatter)
	shortFormatter := interactive.NewMDFormatter(shortLineFormatter, interactive.MdHeaderFormatter)

	return &Teams{
		log:             log,
		executorFactory: executorFactory,
		reporter:        reporter,
		botName:         cfg.BotName,
		ClusterName:     clusterName,
		AppID:           cfg.AppID,
		AppPassword:     cfg.AppPassword,
		Notification:    cfg.Notification,
		bindings:        cfg.Bindings,
		commGroupName:   commGroupName,
		MessagePath:     msgPath,
		Port:            port,
		conversations:   make(map[string]conversation),
		botMentionRegex: botMentionRegex,
		longFormatter:   longFormatter,
		shortFormatter:  shortFormatter,
	}, nil
``

New funcs instead of `mdLineFormatter`:
```go
// longLineFormatter represents new line formatting for MS Teams where message has multiple sections.
// Unfortunately, it's different from all others integrations.
func longLineFormatter(msg string) string {
	// e.g. `:rocket:` is not supported by MS Teams, so we need to replace it with actual emoji
	msg = replaceEmojiTagsWithActualOne(msg)
	return fmt.Sprintf("%s<br>", msg)
}

func shortLineFormatter(msg string) string {
	// e.g. `:rocket:` is not supported by MS Teams, so we need to replace it with actual emoji
	msg = replaceEmojiTagsWithActualOne(msg)
	return fmt.Sprintf("%s\n", msg)
}
```

Updated `convertInteractiveMessage` func:
```go
func (b *Teams) convertInteractiveMessage(in interactive.Message) string {
	if in.HasSections() {
		// MS Teams doesn't respect multiple new lines, so it needs to be rendered
		// with `<br>` tags instead  ¯\_(ツ)_/¯
		return interactive.MessageToMarkdown(b.longFormatter, in)
	}
	return interactive.MessageToMarkdown(b.shortFormatter, in)
}
```

the bug was unfortunately introduced here: https://github.com/kubeshop/botkube/pull/737/files#diff-dab5cea33bef23767be4be070914501c91d73cab9748905293abb4bc47b7b97c

Reply:
Sure thing, we'll do that 👍